listenbrainz/webserver/views/api.py

```diff
@@ -348,6 +349,45 @@ def validate_token():
         })
 
 
+@api_bp.route('/delete-listen', methods=['POST'])
+@crossdomain(headers="Authorization, Content-Type")
+@ratelimit()
+def delete_listen():
+    """ 
+    Delete a particular listen from the currently logged-in user's listen history.
```

Comment:
Will this go in the public docs? We should make it better if so.

Reply:
Add what parameters it takes, what it returns etc.

Reply:
done